### PR TITLE
fix(components): Fix InfinteScroll triggered on open/close animation.

### PR DESCRIPTION
**Pull request by tajespasarela**
Set prettier@2.4.1 version.

[EX-5347]

What this PR does is an update to the last XComponents alpha version. Changing the removed animations to new animations factories, and also set prettier version to `2.4.1` to avoid long `class` attribute in one line.

[EX-5347]: https://searchbroker.atlassian.net/browse/EX-5347?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/mobile/mobile.vue [hunk 1]
```diff
@@ -50,7 +51,14 @@
       <!-- eslint-disable max-len-->
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply src/components/mobile/mobile.vue:51]: done

File: package.json [hunk 2]
```diff
@@ -71,6 +71,7 @@
     "eslint": "7.18.0",
     "http-server": "^14.1.0",
     "lint-staged": "~10.0.7",
+    "prettier": "^2.4.1",
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply package.json]: done

**Review by LuisMartinez15 — DISMISSED** [review]
LGTM

**Review by CachedaCodes — APPROVED** [review]

**Review by LuisMartinez15 — APPROVED** [review]
LGTM

**Pull request merged**